README.adoc

```diff
@@ -6,18 +6,47 @@ Peter Lawrey
 image:https://maven-badges.herokuapp.com/maven-central/net.openhft/chronicle-threads/badge.svg[link="https://search.maven.org/artifact/net.openhft/chronicle-threads/2.20.104/jar"]
 image:https://javadoc.io/badge2/net.openhft/chronicle-threads/javadoc.svg[link="https://www.javadoc.io/doc/net.openhft/chronicle-threads/latest/index.html"]
 
+
 == Thread pool
 
-This library provide a high performance thread pool.
-This thread pool is design to share blocking, monitoring and busy waiting threads.
-Busy waiting tasks can be prioritised for HIGH, MEDIUM, DAEMON (low priority) as well as TIMER (fixed rate events) tasks in a single thread without creating garbage.
+This library provides a high performance thread pool that shares blocking, monitoring, and busy waiting threads.
+Busy waiting tasks have `HIGH`, `MEDIUM`, or `DAEMON` (low) priority, or `TIMER` (fixed rate events) status. All tasks run on a single thread without creating garbage.
+
+See `net.openhft.chronicle.core.threads.EventLoop` and `net.openhft.chronicle.threads.EventGroup`.
+
+
+== Monitoring the event loop
+
+The `MonitorEventLoop` thread monitors supplier threads to make sure event loop latency remains within acceptable bounds. The thread monitors latency by measuring the time the `action` method in the `ThreadMonitorEventHandler` takes to run. Whenever a supplier thread runs beyond an acceptable latency limit, the system prints a stack trace.
+
+Set the monitor event interval with system property `MONITOR_INTERVAL_MS` from the `EventGroup` class:
+
+[source,java]
+----
+private static final long MONITOR_INTERVAL_MS = Long.getLong("MONITOR_INTERVAL_MS", 100);
+----
+
+Disable the monitor by setting the system property:
+
+[source,java]
+----
+disableLoopBlockMonitor=False
+----
+
+Use any stack trace information to improve the design for efficiency.
+
+=== Recommendations:
+
+- Impose an interval of 100µs for every event loop.
+- Code out of an unacceptable delay by returning quickly.
+
 
-See `net.openhft.chronicle.core.threads.EventLoop` and `net.openhft.chronicle.threads.EventGroup`
 
 == Pauser
 
 Chronicle Threads provides a number of implementations of the `net.openhft.chronicle.threads.Pauser` interface.
-The canonical way to make use of `Pauser` is below:
+
+The canonical way to use `Pauser`:
```

Comment:
I'm not sure I would use the word "canonical", I think we should use the word "recommended" or "default? as "canonical" is usually not a word that is used in IT.

Reply:
Agree, overused in my experience.

Reply:
Amended.